### PR TITLE
Rails 5 mongoid 6

### PR DESCRIPTION
**Pull request by boie0025**
Changes for Mongoid 6,update readme, pull in older fixes for misc bugs and green specs on rails 5.0.x and mongoid 6.0.x

### CODE REVIEW

File: activeadmin-mongoid.gemspec [hunk 1]
```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |gem|
   gem.version       = ActiveAdmin::Mongoid::VERSION
   gem.license       = 'MIT'
 
-  gem.add_runtime_dependency 'mongoid',     ['> 3.0', '< 5.0']
-  gem.add_runtime_dependency 'activeadmin', ['>= 1.0.0.pre', '< 2']
+  gem.add_runtime_dependency 'mongoid',     ['>= 6.0', '< 6.1']
+  gem.add_runtime_dependency 'activeadmin', ['>= 1.0.0.pre5', '< 2']
```
Comment (elia) [inline activeadmin-mongoid.gemspec]: @boie0025 what about just using `~> 1.0` here?

Comment (boie0025) [inline activeadmin-mongoid.gemspec]: Good call

**Comment by varyonic** [comment]
@boie0025 Are you going to merge?

**Comment by boie0025** [comment]
Yes, doing a bit more testing in our on this before we pull it into master.  Will merge and release soon.

**Pull request merged**